### DISCUSSION TITLE
How to interpret this plot_components results?

### DISCUSSION THREAD

**Discussion by landkwon94** [ourownstory/neural_prophet#1429] [Q&A - get help using NeuralProphet]
![Figure_1](https://github.com/ourownstory/neural_prophet/assets/73383617/73c8309d-0952-440f-b00b-6d32e9aac37e)

This is the plot after training the Neural Prophet model in my case.

I understood the meaning of weights, which implies how lagged days affect forecasting results.

But I still do not perfectly understood what these 'lagged regressors' in the y-axis exactly mean. Although I read paper several times, I cannot fully understand..!

Can you give me some comments?

Many thanks!

**Comment by leoniewgnr** [comment]
So basically the forecast is a sum of all components. Because the lagged regressors are different for each step in the forecast, there are multiple lagged regressors displayed in varying colors (different shades of blue). They basically represent the weight ie the lagged regressor plus the other components result in the forecast. If you only forecast one step ahead, this can be seen more clearly. Hope this helps!
